distribution/packages/src/deb/debian/postinst

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Copyright Wazuh Indexer Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The Wazuh Indexer Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+# deb wazuh-indexer postinst script
+
+set -e
+
+echo "Running Wazuh Indexer Post-Installation Script"
+
+product_dir=/usr/share/wazuh-indexer
+config_dir=/etc/wazuh-indexer
+data_dir=/var/lib/wazuh-indexer
+log_dir=/var/log/wazuh-indexer
+pid_dir=/var/run/wazuh-indexer
+
+# Apply Security Settings
+if [ -d ${product_dir}/plugins/opensearch-security ]; then
+    bash ${product_dir}/plugins/opensearch-security/tools/install_demo_configuration.sh -y -i -s > ${log_dir}/install_demo_configuration.log 2>&1
```

Comment:
We have deleted this file in #106

Reply:
It would be great if we could initialize the wazuh-indexer cluster here, running `/usr/share/wazuh-indexer/bin/indexer-init.sh`